android/build.gradle

```diff
@@ -25,7 +20,6 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:7.2.1'
```

Comment:
Should this be `classpath("com.android.tools.build:gradle:7.3.1")`?

Reply:
yup, thanks! updated